projects/image_chat/transresnet_multimodal/transresnet_multimodal.py

```diff
@@ -423,13 +423,17 @@ def report(self):
         """
         Report per-dialogue round metrics.
         """
-        m = {k: {} for k in ["first_round", "second_round", "third_round+"]}
+        m = {}
```

Comment:
oh god - getting into the weeds of the individual projects now. I suppose people will know if their project hasn't been converted because it'll stop working?